python/ray/worker.py

```diff
@@ -566,15 +565,7 @@ def get_resource_ids():
         each pair consists of the ID of a resource and the fraction of that
         resource reserved for this worker.
     """
-    worker = global_worker
-    worker.check_connected()
-
-    if _mode() == LOCAL_MODE:
-        raise RuntimeError(
-            "ray.worker.get_resource_ids() currently does not work in local_mode."
-        )
-
-    return global_worker.core_worker.resource_ids()
+    raise DeprecationWarning("get_resource_ids() is deprecated")
```

Comment:
Is there any public alternative we should suggest here? @ericl

Reply:
get_gpu_ids() I think (cc @wuisawesome )

Reply:
Yup, GPUs are the only resource with ids. (Changing this config isn't a public api) https://sourcegraph.com/github.com/ray-project/ray/-/blob/src/ray/common/ray_config_def.h?L550